rpc/testservice_test.go

```diff
@@ -196,7 +196,6 @@ func (s *notificationTestService) SomeSubscription(ctx context.Context, n, val i
 			}
 		}
 		select {
-		case <-notifier.Closed():
 		case <-subscription.Err():
 		}
```

Comment:
```
rpc/testservice_test.go:198:3: S1000: should use a simple channel send/receive instead of `select` with a single case (gosimple)
		select {
		^
```